py4DSTEM/process/diffraction/crystal.py

```diff
@@ -108,7 +109,16 @@ def __init__(
         
         # Calculate lattice parameters
         self.calculate_lattice()
-        
+    
+    # TODO check for and copy all important attributes
+    def copy(self):
```

Comment:
Do we expect users to do this, if not we should name it `_copy()` as it's not really a copy but init a new crystal,